tests/e2e/framework/multicluster.go

```diff
@@ -105,6 +105,7 @@ func (k *KubeInfo) generateRemoteIstio(dst string, useAutoInject bool, proxyHub,
 			log.Infof("Endpoint for service %s not found", svc)
 		}
 	}
+	helmSetContent += " --set security.selfSigned=false"
```

Comment:
Can you explain (add comment) why we need to have selfSigned false?

Reply:
Sure. Once/If it's out of WIP